### PR TITLE
Also lint README.md using markdownlint-cli2-action

### PR DESCRIPTION
**Pull request by georgeguimaraes**
Is this a good idea?

### CODE REVIEW

**Review by josevalim — COMMENTED** [review]
@whatyouhide decides. :)

File: .markdownlint.jsonc [hunk 1]
```diff
@@ -30,8 +30,12 @@
       "script"
     ]
   },
+  // This warns if emails are used without '<' and '>'. That's ok.
```
Comment (paulo-ferraz-oliveira) [inline .markdownlint.jsonc:33]:
```suggestion
  // This warns if URLs or emails are used without '<' and '>'. That's ok.
```

File: .markdownlint.jsonc [hunk 2]
```diff
@@ -30,8 +30,12 @@
...
+  "MD034": false,
   // This warns if you have spaces in code blocks. Sometimes, that's fine.
```
Comment (paulo-ferraz-oliveira) [inline .markdownlint.jsonc:35]:
```suggestion
  // This warns if you have spaces in code spans. Sometimes, that's fine.
```

Comment (paulo-ferraz-oliveira) [reply .markdownlint.jsonc:35]: spans =/= blocks. In code blocks it's Ok to have white space.

File: .markdownlint.jsonc [hunk 3]
```diff
@@ -30,8 +30,12 @@
...
   "MD038": false,
+  // This warns if there's no H1 at the first line. Disabled because of README.md.
+  "MD041": false,
```
Comment (whatyouhide) [inline .markdownlint.jsonc:38]: @paulo-ferraz-oliveira does this get fixed if we wrap the first `` in the README inside a `<h1>`?

File: .markdownlint.jsonc (hunk 1 above)
Comment (whatyouhide) [reply .markdownlint.jsonc:33]: Is it ok? Can't we fix problematic links in the README instead?

Comment (georgeguimaraes) [reply .markdownlint.jsonc:33]: Sure, let me fix the emails in README then

File: .markdownlint.jsonc (hunk 3 above)
Comment (georgeguimaraes) [reply .markdownlint.jsonc:38]: @whatyouhide Tried that, but then GitHub would insert the heading border twice. There's no way to hide one the h1 depending on the dark/light theme.

Comment (georgeguimaraes) [reply .markdownlint.jsonc:38]: omg, I could try to wrap both imgs in the same h1 🤦🏻

**Comment by georgeguimaraes** [comment]
@whatyouhide pls take a look again

**Comment by whatyouhide** [comment]
Fantastic, thanks @georgeguimaraes 🙏

**Comment by georgeguimaraes** [comment]
CleanShot 2023-10-31 at 09 41 16

It didn't work as expected. I'll try to fix it now.

**Comment by georgeguimaraes** [comment]
Done in https://github.com/elixir-lang/elixir/commit/d1f493cff58eef21c9e08884340c5005663b58a5.

**Pull request merged**